### PR TITLE
added note about extended date

### PR DESCRIPTION
**Pull request by rkelson**

### CODE REVIEW

File: _posts/2020-05-13-2020-utah-lidar-acquisition.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ tags:
...
+AGRC has awarded Aero-Graphics, Inc. and The Sanborn Map Company, Inc. to acquire [**20,958 square miles**](http://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar elevation data. The lidar will be a combination of USGS Quality Level 1 (_QL1_), 8 points per square meter and Quality Level 2 (_QL2_), 2 points per square meter depending on the area. The 2019 QL2 project covering portions of Kane County, Utah was acquired last fall, is going through USGS Quality Assurance, and is scheduled to be completed by June 30, 2020. ( _Due to Covid-19 related setbacks deliverables shoudd available by November 2020._ ) The 2020 acquisitions have begun and will continue throughout the year. These project areas will be completed in 2021 and project deliverables are scheduled to be completed by June 30, 2021.
```
Comment (steveoh) [inline _posts/2020-05-13-2020-utah-lidar-acquisition.md:14]: **nitpick**: always use https links
**chore**: spelling fix
```suggestion
AGRC has awarded Aero-Graphics, Inc. and The Sanborn Map Company, Inc. to acquire [**20,958 square miles**](https://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar elevation data. The lidar will be a combination of USGS Quality Level 1 (_QL1_), 8 points per square meter and Quality Level 2 (_QL2_), 2 points per square meter depending on the area. The 2019 QL2 project covering portions of Kane County, Utah was acquired last fall, is going through USGS Quality Assurance, and is scheduled to be completed by June 30, 2020. ( _Due to Covid-19 related setbacks deliverables should available by November 2020._ ) The 2020 acquisitions have begun and will continue throughout the year. These project areas will be completed in 2021 and project deliverables are scheduled to be completed by June 30, 2021.
```

File: _posts/2020-05-13-2020-utah-lidar-acquisition.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ tags:
...
 
 The Federal Emergency Management Agency, the U.S. Forest Service, and the U.S. Geological Survey (USGS) National Map 3D Elevation Program (3DEP) are also acquiring [**13,681 square miles**](http://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar in Utah this year. The lidar will mostly be QL2, with a few small areas as QL1. The acquisitions will begin in the fall of 2020 and project deliverables are scheduled to be completed by June 30, 2022.
```
Comment (steveoh) [inline _posts/2020-05-13-2020-utah-lidar-acquisition.md:16]: **nitpick**: always use https links
```suggestion
The Federal Emergency Management Agency, the U.S. Forest Service, and the U.S. Geological Survey (USGS) National Map 3D Elevation Program (3DEP) are also acquiring [**13,681 square miles**](https://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar in Utah this year. The lidar will mostly be QL2, with a few small areas as QL1. The acquisitions will begin in the fall of 2020 and project deliverables are scheduled to be completed by June 30, 2022.
```

**Review by steveoh — CHANGES_REQUESTED** [review]
i found some issues that slipped through and a new spelling error.

**Comment by rkelson** [comment]
is there something else I need to do?

**Review by steveoh — APPROVED** [review]

**Pull request merged**